### DISCUSSION TITLE
Feishin not pulling artist art/images from Navidrome

### DISCUSSION THREAD

**Discussion by VoidJuiceConcentrate** [jeffvli/feishin#792] [Q&A] [answered]
Hello all! I'm wondering if I didn't do something right here, when showing the list of artists (screenshot below) Feishin does not appear to be pulling Artist images. It works fine for Album art and individual song art. Um, the artist art shows fine in Navidrome's built-in UI, let me know if theres something I may have left out that's needed. 

![image](https://github.com/user-attachments/assets/fdbb9b90-b94d-4c4b-b6f6-b432326594c9)

**Comment by jeffvli** [comment, marked as answer]
Go to the settings page, scroll to the bottom, press clear on both cache options and restart the app.

**Comment by VoidJuiceConcentrate** [comment]
Hello! Thank you for getting back to me. 

I'm using the Docker instance for Feishin, clearing the internal cache did not work. But when I cleared my browsers cache, that made it work instantly. Thank you for the help!
